### DISCUSSION TITLE
请问一下这个接口返回值是怎么写的？

### DISCUSSION THREAD

**Discussion by nsnsqk** [ddiu8081/chatgpt-demo#446] [Q&A]
我发现虽然接口是流式返回的，但是接口最终结果确实一个最终的输出内容。是怎么实现的呢？翻看了源码，看着也是正常的流式调用，没发现什么手法在里面。有没有大佬指点一下，谢啦

**Comment by wzmtgggg** [comment]
> 我发现虽然接口是流式返回的，但是接口最终结果确实一个最终的输出内容。是怎么实现的呢？翻看了源码，看着也是正常的流式调用，没发现什么手法在里面。有没有大佬指点一下，谢啦
知道了不兄弟👬👬👬👬👬👬👬
